core/Pos/Core/Types.hs

```diff
@@ -103,6 +103,12 @@ import           Pos.Data.Attributes        (Attributes)
 -- Address
 ----------------------------------------------------------------------------
 
+-- | Hash of address
```

Comment:
It's not hash of address, this hash is used inside `Address`, but it's applied to `PublicKey`.